brainscore_vision/models/effnetb1_cutmixpatch_SAM_robust32_avge6e8e9e10_manylayers_324x288/model.py

```diff
@@ -1,7 +1,7 @@
 import functools
 
 import torch
-from brainscore_vision.model_helpers.activations import PytorchWrapper, KerasWrapper

```

Comment:
this model imported the `KerasWrapper` just for jokes?

Reply:
exactly :)